Netw4ppl/app/src/main/java/netw4ppl/ines/AddPersonActivity.java

```diff
@@ -197,7 +199,7 @@ public static int getNextId(String key) {
      */
     public boolean verificationInputPerson() {
         boolean all_good = true;
-
+        System.out.println("verificationInputPerson");
```

Comment:
TO be deleted before merging to preprod. Maybe you should add a tag to remember

Reply:
👍🏼